gnss_lib_py/utils/sv_models.py

```diff
@@ -0,0 +1,611 @@
+"""Model GNSS SV states (positions and velocities).
+
+Functions to calculate GNSS SV positions and velocities for a given time.
+"""
+
+__authors__ = "Ashwin Kanhere, Bradley Collicott"
+__date__ = "17 Jan, 2023"
+
+import warnings
+import numpy as np
+
+import gnss_lib_py.utils.constants as consts
+from gnss_lib_py.utils.coordinates import ecef_to_el_az
+from gnss_lib_py.parsers.navdata import NavData
+from gnss_lib_py.parsers.ephemeris import EphemerisManager
+from gnss_lib_py.utils.time_conversions import gps_millis_to_tow, gps_millis_to_datetime
+
+
+def svs_from_el_az(elaz_deg):
```

Comment:
Can I move this to `coordinates.py` where we have the *almost* opposite of this already (`ecef_to_el_az`)?